dbbench.go

```diff
@@ -122,6 +126,34 @@ func main() {
 	if *baseDir == "" {
 		*baseDir = filepath.Dir(configFile)
 	}
+	if connectionUrl != nil {
+		u, err := url.Parse(*connectionUrl)
+		if err != nil {
+			log.Fatalf("parsing database url %v", err)
+		}
+		if u.Scheme != "" {
+			*driverName = u.Scheme
+		}
+		if u.Host != "" {
+			GlobalConfig.Host = u.Host
+		}
+		if u.User.Username() != "" {
+			GlobalConfig.Username = u.User.Username()
+		}
+		pass, isPassSet := u.User.Password()
+		if isPassSet {
+			GlobalConfig.Password = pass
+		}
+		if u.Hostname() != "" {
+			GlobalConfig.Host = u.Hostname()
+		}
+		if u.Port() != "" {
+			GlobalConfig.Port, _ = strconv.Atoi(u.Port())
```

Comment:
Please do something useful on error handling here -- probably warn the user

Reply:
@awreece , about the error handling. I think that no error can occur here, according to `Port()` definition:

Reply:
@awreece , in addition please note that the driverName is not manage with flag.Var, so why not manage the url (which contains the driver name) the same way?

Reply:
Thanks for showing `flag.Func()`, I agree that its better!